display_list/dl_op_spy.cc

```diff
@@ -0,0 +1,171 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "flutter/display_list/dl_op_spy.h"
+
+namespace flutter {
+
+bool DlOpSpy::DidDraw() {
```

Comment:
I've been using lower case for getters which I think is compliant with the style guidelines - `did_draw()`...